plugins/3MFWriter/SettingsExportModel.py

```diff
@@ -117,6 +117,9 @@ def _exportSettings(settings_stack):
         is_exportable = any(key in SettingsExportModel.PER_MODEL_EXPORTABLE_SETTINGS_KEYS for key in user_keys)
 
         for setting_to_export in user_keys:
+            show_in_menu = True
+            if setting_to_export  in SettingsExportModel.PER_MODEL_EXPORTABLE_SETTINGS_KEYS:
+                show_in_menu = False
```

Comment:
expression can be simplified to
```suggestion
            show_in_menu = setting_to_export not in SettingsExportModel.PER_MODEL_EXPORTABLE_SETTINGS_KEYS
```